tests/playTests.sh

```diff
@@ -537,6 +537,29 @@ fi
 
 rm tmp*
 
+$ECHO "\n**** zstd --list/-l single frame tests ****"
+./datagen > tmp1
+./datagen > tmp2
+./datagen > tmp3
+./datagen > tmp4
+$ZSTD tmp*
+$ZSTD -l *.zst
+$ZSTD -lv *.zst
+$ZSTD --list *.zst
+$ZSTD --list -v *.zst
+
```

Comment:
Add a test that check the detected error regarding return code on mixed `zst` and `non-zst` input.
Typically something like : 
`!$ZSTD -l tmp tmp.zst`.

The test should fail with older code `error = FIO_listFile()`
and work with `error |= FIO_listFile()`.